src/controllers/User.js

```
@@ -0,0 +1,73 @@
+import { userValidate, validate } from '../validation';
+import UserModel from '../models/User';
+import {encryptPassword, signToken, decryptPassword}  from '../helpers/auth';
+import asyncHandler  from '../middlewares/async';
+import Response from '../utils';
+
+
+export const signup = asyncHandler(async (req, res) => {
+
+  const { image, imageId } = req;
+  const data = { ...req.body, image, imageId };
+
+  const { details: errors } = validate(userValidate.CreateSchema, data);
+  if (errors)
+    return Response.error(
+      res,
+      400,
+      `Please provide ${errors[0].context.key}`,
+      errors[0]
```

you do not need to return this technical error to the user